test/api/test-multithread.c

```diff
@@ -93,8 +93,11 @@ thread_func (void *data)
 }
 
 static void
-test_body (void)
+test_body (const char *backend)
 {
+  if (!hb_font_set_funcs_using (font, backend))
+    return;
```

Comment:
Do we really want to silently ignore the return status? I does not look like it should ever happen.

Reply:
It's the coretext backend for example wouldn't load a PingFangUI font not at its default location. But yeah in the tests we can assert.

Reply:
I mean setting the font functions should not fail, that using the new face can fail is orthogonal to this.

Reply:
But setting font funcs do fail. Not with the fonts the test is using though. So I'll make it an assert here.